modules/core/arrow-test/src/main/kotlin/arrow/test/laws/EqKLaws.kt

```diff
@@ -12,13 +13,13 @@ object EqKLaws {
   fun <F> laws(
     EQK: EqK<F>,
     EQ: Eq<Kind<F, Int>>,
```

Comment:
This EQ instance is no longer needed as `EqK` provides all information needed

Reply:
ok, will drop the last test in Laws then too